### PR TITLE
Update de.json

### PR DESCRIPTION
**Pull request by FloriF**
Translated Publish and Workspaces Sections.
Added some missing translations.
Typos.

Also, unified some things:
'Zeigt...' instead of 'Zeige...' or '... zeigen' in the settings menu descriptions.
'Vault' is translated as 'Vault' (as a obsidian-specific term)
Replaced 'CTRL' with 'STRG'
'Panel' is translated as 'Fensterbereich' instead of 'Bereich'
'App' is now translated as 'Anwendung'
'Verknüpfung, Verlinkung, Link' is now always 'Verlinkung'

---

As far as I can see, everything is now translated - yay!

### CODE REVIEW

File: de.json [hunk 1]
```diff
@@ -486,8 +486,8 @@
 		},
 		"markdown-format-importer": {
 			"name": "Markdown Format Importprogramm",
-			"desc": "Konvertiert das Markdown von anderen Applikationen in das Obsidian Format.",
-			"action-open": "Öffne das Markdonw Importprogramm",
+			"desc": "Konvertiert das Markdown von anderen Anwendungen in das Obsidian Format.",
+			"action-open": "Öffne das Markdon Importprogramm",
```
Comment (auxilium-systems) [inline de.json:490]: "Markdown" instead of "Markdon"

File: de.json [hunk 2]
```diff
@@ -15,53 +15,53 @@
 			"option-strict-line-break": "Strikte Zeilenbegrenzung",
 			"option-strict-line-break-description": "Im Vorschau-Modus werden einzelne Zeilenumbrüche gemäß der Markdown-Spezifikationen ignoriert. Schalte dies aus, um einzelne Zeilenumbrüche im Vorschau-Modus anzuzeigen.",
 			"option-show-frontmatter": "Frontmatter anzeigen",
-			"option-show-frontmatter-description": "Frontmatter im Vorschaumodus als Quellcode Block anzeigen. Ausschalten um Frontmatter Bereich zu deaktivieren.",
-			"option-use-markdown-links": "Markdown links benutzen",
-			"option-use-markdown-links-description": "Linkvorschlag vervollständigt Markdown-Links und Bilder automatisch anstelle von Wiki-Links.",
+			"option-show-frontmatter-description": "Zeigt den Frontmatter im Vorschaumodus als Code-BLock. Schalte dies aus, um den Frontmatter Bereich zu deaktivieren.",
+			"option-use-markdown-links": "Markdown-Links benutzen",
+			"option-use-markdown-links-description": "Vervollständigt automatisch Links und Bilder mit Markdown-Links anstelle von Wikki-Links.",
```
Comment (auxilium-systems) [inline de.json:20]: "Wiki" instead of "Wikki"

**Review by auxilium-systems — COMMENTED** [review]
Super Übersetzungen, die Änderungen finde ich sehr gelungen.

**Comment by ericaxu** [comment]
Are we waiting for additional commits to resolve the comments?

**Comment by FloriF** [comment]
I added these changes, I was going to wait for the next update to check for further typos - but this works too^^'

**Comment by ericaxu** [comment]
Thank you for your work!

**Pull request merged**